osv/third_party/univers/alpine.py

```diff
@@ -5,6 +5,7 @@
 # Visit https://aboutcode.org and
 # https://github.com/nexB/univers for support and download.
 import attr
+import re
```

Comment:
Can you add a comment here describing the changes you made? i.e. supporting revisions?